client/my-sites/plans/main.jsx

```diff
@@ -184,6 +169,7 @@ const Plans = React.createClass( {
 
 						{ ! hasJpphpBundle && this.showMonthlyPlansLink() }
 						<QueryPlans />
+						<QuerySitePlans />
```

Comment:
This is working currently, since `SiteNotice` also uses the `QuerySitePlans` component.